src/sentry/taskworker/namespaces.py

```diff
@@ -24,6 +24,8 @@
 
 digests_tasks = taskregistry.create_namespace("digests")
 
+export_tasks = taskregistry.create_namespace("export")
```

Comment:
Should we bump up the processing deadlines on this namespace? Based on our metrics, export tasks can run upwards of 15s.